src/static/riot/competitions/editor/form.tag

```diff
@@ -97,6 +101,9 @@
                 <button class="ui basic red button discard" onclick="{ discard }">
                     Discard Changes
                 </button>
+                <button class="ui primary button">
+                    <a style="color: #fff" href="{URLS.COMPETITION_DETAIL(opts.competition_id)}">Back To Competition</a>
```

Comment:
avoid inline styling

Reply:
we shouldn't need a style attribute at all. if we remove the button tag altogether, and swap out the style attr on the `a` tag with `class="ui primary button"` it will style the anchor tag as a button correctly with white text and maintain the navigation functionality.

Reply:
Right on, ill make those changes now :)